spec/view_changes_need_screenshots_spec.rb

```diff
@@ -39,9 +39,18 @@ module Danger
         expect(@dangerfile.status_report[:warnings]).to be_empty
       end
 
-      it 'does nothing when a PR with view code changes has a screenshot defined with a html tag' do
+      it 'does nothing when a PR with view code changes has a screenshot defined with a html tag with different attributes before src' do
         allow(@plugin.github).to receive(:pr_body)
-          .and_return("see screenshot:\n<img width=300 src=\"https://github.com/bloom/DayOne-Apple/assets/4780/1f9e01a8-c63d-41d4-9ac8-fa9a5182ab55\"> body body")
+          .and_return("see screenshot:\n<img width=300 hint=\"First screenshots\" src=\"https://github.com/bloom/DayOne-Apple/assets/4780/1f9e01a8-c63d-41d4-9ac8-fa9a5182ab55\"> body body")
```

Comment:
I just realized that our RegEx would fail to detect tags that might have `>` within any of the other string attributes, like `<img alt="a > b" src="…">`…

But tbh I think that's an edge case not worth spending time on, so I'm good with it.